control-plane-operator/controllers/hostedcontrolplane/kubevirt_assets/files/kubeconfig_template.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
you shouldn't need this file anymore if you create the kubeconfig using the hypershift standard way.